lib/AST/Module.cpp

```diff
@@ -988,6 +988,115 @@ ProtocolConformanceRef ModuleDecl::lookupConformance(Type type,
       getASTContext().evaluator, request, ProtocolConformanceRef::forInvalid());
 }
 
+/// Synthesize a builtin tuple type conformance to the given protocol, if
+/// appropriate.
+static ProtocolConformanceRef getBuiltinTupleTypeConformance(
+    Type type, const TupleType *tupleType, ProtocolDecl *protocol) {
+  // Tuple type are Sendable when all of their element types are Sendable.
```

Comment:
Wohoo, that unlocks Void conforming to Sendable as well right? AFAIR at least it was related to tuples hm.

Reply:
Yes, that's why I'm pulling it in.